air-interpreter/src/logger.rs

```diff
@@ -15,7 +15,6 @@
  */
 
 use air_log_targets::TARGET_MAP;
-
```

Comment:
Its nice to have internal/external crates imports separated, so lets keep the blank line